webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -73,7 +63,7 @@ class notificationContainer extends React.Component {
               onClickedLink={clickedLink}
               toggleDrawerHide={toggleDrawer}
               isExpandable={false}
-              translations={translations}
+              translations={translateObject(PanelWrapper.defaultProps.translations)}
```

Comment:
Shouldn't you receive the default props as actual props and translate the props?